client/components/AddBank.jsx

```diff
@@ -22,5 +22,5 @@ const AddBank = ({ userId, dispatch }) => {
 };
 
 export default connect(state => ({
-  userId: state.user.userId,
+  id: state.user.id,
```

Comment:
Might be useful to keep the key here as user_id. This allows handy es6 object literal declarations because it matches the user_id field name in the users schema.